source/utilities.cc

```diff
@@ -207,6 +207,51 @@ namespace WorldBuilder
     }
 
 
+    std::pair<Point<2>, Point<2> >
+    get_bounding_box (const std::vector<Point<2> > &point_list,
+                      std::size_t original_number_of_coordinates,
+                      const double buffer_around_fault)
+    {
+      std::vector<double> x_list(original_number_of_coordinates,0.0);
+      std::vector<double> y_list(original_number_of_coordinates,0.0);
+
+      for (size_t j=0; j<original_number_of_coordinates; ++j)
+        {
+          x_list[j] = point_list[j][0];
+          y_list[j] = point_list[j][1];
+        }
+
+      // Initialize any point to create bounding_box, it will be overwritten later.
+      Point<2> reference_point(point_list[0]);
+      // Corner coordinates of the bounding polygon
+      std::pair<Point<2>, Point<2> > bounding_box (reference_point, reference_point);
```

Comment:
You should be able to just write it like this, and then don't need `reference_point` at all:
```suggestion
      std::pair<Point<2>, Point<2> > bounding_box;
```

Reply:
I tried that, but it complains ```no default constructor exists for class "std::pair<WorldBuilder::Point<2>, WorldBuilder::Point<2>>"```.

Reply:
That is because I would not know what the default constructor would need to be, since a point requires a coordinate system. Although I guess it could be `invalid`.

Reply:
In #306, I initialize a pair of points this way:
```
std::pair<Point<2>, Point<2> > bounding_box =
{{
      -std::numeric_limits<double>::max(),
      -std::numeric_limits<double>::max(),
      -std::numeric_limits<double>::max(),
      cartesian
    },
    {
      +std::numeric_limits<double>::max(),
      +std::numeric_limits<double>::max(),
      +std::numeric_limits<double>::max(),
      cartesian
    }
  }
```